src/tests/ftest/control/version.py

```diff
@@ -56,25 +36,21 @@ def test_version(self):
         :avocado: tags=DAOSVersion,test_version
         """
         # Get RPM version.
-        rpm_command = "rpm -qa|grep daos-server"
+        rpm_command = "rpm -qa | grep daos-server"
         output = run_pcmd(hosts=self.hostlist_servers, command=rpm_command)
         self.log.debug("RPM output = %s", output)
         rc = output[0]["exit_status"]
         stdout = output[0]["stdout"]
         if rc != 0:
-            msg = "DAOS RPMs not properly installed: rc={}".format(rc)
-            self.append_error(msg, stdout)
-            self.report_result()
+            report_errors(self, ["DAOS RPMs not properly installed: rc={}".format(rc)])
         rpm_version = None
         for rpm in stdout:
             result = re.findall(r"daos-server-[tests-|tests_openmpi-]*([\d.]+)", rpm)
             if result:
                 rpm_version = result[0]
                 break
         if not result:
-            msg = "RPM version could not be defined"
-            self.append_error(msg, stdout)
-            self.report_result()
+            report_errors(self, ["RPM version could not be defined"])
         self.log.info("RPM version = %s", rpm_version)
 
         # Remove configuration files
```

Comment:
@phender , as indicated in the first PR, I would like to keep these cleanup actions.
Indeed, the /etc/daos configuration files seems to not be properly cleaned between each tests and configuration files such as certificates are not cleaned.
```
22:22:59 INFO | Command: sudo find /etc/daos/certs -type f -delete -print
Results:
  wolf-105vm2: exit_status=0, interrupted=False:    /etc/daos/certs/server.crt
    /etc/daos/certs/clients/agent.crt
    /etc/daos/certs/server.key
    /etc/daos/certs/daosCA.crt
```